docs/source/user-guide/vision/formatter_objects.rst

```diff
@@ -73,9 +73,75 @@ For object detection tasks, it is required to implement the following formatters
 
 - **Detections Predictions** - should implement an instance of :class:`detection_formatters.DetectionPredictionFormatter`.
 
+
 Converting From Common Formats
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-TBD (Not implemented yet)
+
+To initialize `DetectionLabelFormatter` for common label formats the notion of the bbox format notation can be used:
+
+.. code-block:: python
+
+    import torch
+    from deepchecks.vision.utils import DetectionLabelFormatter
+
+    yolo_formatter = DetectionLabelFormatter('lcxcywhn') 
+    pascal_formatter = DetectionLabelFormatter('lxyxy')
+    coco_formatter = DetectionLabelFormatter('lxywh')
+    albumentations_formatter = DetectionLabelFormatter('lxyxyn')
+
+By using provided bbox format notation deepchecks will know how to transform labels to the format required by deepchecks, which is 'lxywh'.
+
+Elements of the bbox format notation:
+    * l - label (class) of the object;
+    * xy - top-left or bottom-right corner of the bbox. (First 'xy' element is always interpreted as x-min, y-min, and second as x-max, y-max). Coordinates of the bounding box's corners are represented with respect to the top-left corner of the image which has (x, y) coordinates (0, 0);
+    * cxcy - center of the bbox;
+    * wh - width and height of the bbox;
+    * n - element placed at the beggining or at the end of the notation indicates whether coordinates are normalized or not. Which means that coordinates of the bbox are representated not in pixels but as ration of x / image-width and y / image-height.
+
+An important moment to note is that `DetectionLabelFormatter` do not work with labels represented by JSON, XML, or python dictionaries.
+The acceptable objects types for `DetectionLabelFormatter` are numpy arrays, pytorch tensors, and python sequence-like objects.
```

Comment:
```suggestion
The label batch processed by `DetectionLabelFormatter` must be an iterable of numpy arrays.
```
Does the code really work on label batches which are iterables of tensors or other stuff?

Reply:
> Does the code really work on label batches which are iterables of tensors or other stuff?

do not know what you mean by `other stuff` but yes it works with pytorch tensors